hazelcast-all/pom.xml

```diff
@@ -152,7 +152,7 @@
         <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast-aws</artifactId>
-            <version>3.1</version>
+            <version>3.2</version>
```

Comment:
@Serdaro 3.2 support AWS Fargate so it could be good to mention in release notes.
It was a request by community. https://github.com/hazelcast/hazelcast-aws/issues/86